src/components/boardings/boarding-edit/boarding-edit.component.js

```diff
@@ -62,11 +60,9 @@ class BoardingEditPage extends Component {
     this.validateSchema();
     boardingService.updateBoarding(this.dataObject).then((result) => {
       if (this.state.isNew) {
-        history.push(VIEW_BOARDING_PAGE.replace(":id", result.insertedId));
+        goBoardingViewPage(result.insertedId);
       } else {
-        history.push(
-          VIEW_BOARDING_PAGE.replace(":id", this.state.dataObject._id)
-        );
+        goBoardingViewPage(this.state.dataObject._id);
```

Comment:
reuse shared method in `getData`

Reply:
applies to all the other changes to `boardings` pages
that manually used `VIEW_BOARDING_PAGE` before